Documentation/contributor-guide/contributor-membership.md

```diff
@@ -0,0 +1,151 @@
+below.
+
+## Member
+
+Members are continuously active contributors in the community.  They can have
+issues and PRs assigned to them. Members are expected to remain active 
+contributors to the community.
+
+**Defined by:** Member of the etcd GitHub organization
+
+### Requirements
+
+- Have made multiple contributions to the project or community.  Contribution may include, but is not limited to:
+    - Authoring or reviewing PRs on GitHub. At least one PR must be **merged**.
+    - Filing or commenting on issues on GitHub
+    - Contributing to community discussions (e.g. meetings, Slack, email discussion
+      forums, Stack Overflow)
+- Subscribed to [etcd-dev@googlegroups.com]
+- Have read the [contributor guide]
+- Sponsored by one active maintainer.
+    - Sponsors must be from multiple member companies to demonstrate integration across community.
+    - With no objections from other maintainers
+- Reviewers can be removed by a supermajority of the maintainers or can resign by notifying
+  the maintainers.
+
+### Responsibilities and privileges
+
+- Responsive to issues and PRs assigned to them
+- Granted "triage access" to etcd project
+- Active owner of code they have contributed (unless ownership is explicitly transferred)
+    - Code is well tested
+    - Tests consistently pass
+    - Addresses bugs or issues discovered after code is accepted
+
+**Note:** members who frequently contribute code are expected to proactively
+perform code reviews and work towards becoming a *reviewer*.
+
+## Reviewers
+
+Reviewers are contributors who have demonstrated greater skill in
+reviewing the code from other contributors. They are knowledgeable about both 
+the codebase and software engineering principles. Their LGTM counts towards
+merging a code change into the project. A reviewer is generally on the ladder towards
+maintainership. 
+
+**Defined by:** *reviewers* entry in the [MAINTAINERS] file.
+
+### Requirements
+
+- member for at least 3 months
+- Primary reviewer for at least 5 PRs to the codebase
+- Reviewed or merged at least 20 substantial PRs to the codebase
+- Knowledgeable about the codebase
+- Sponsored by two active maintainer.
+    - Sponsors must be from multiple member companies to demonstrate integration across community.
+    - With no objections from other maintainers
+- Reviewers can be removed by a supermajority of the maintainers or can resign by notifying
+  the maintainers.
+
+### Responsibilities and privileges
+
+- Code reviewer status may be a precondition to accepting large code contributions
+- Responsible for project quality control via code reviews
+    - Focus on code quality and correctness, including testing and factoring
+    - May also review for more holistic issues, but not a requirement
+- Expected to be responsive to review requests
+- Assigned PRs to review related to area of expertise
+- Assigned test bugs related to area of expertise
+- Granted "triage access" to etcd project
+
+## Maintainers
```

Comment:
Just curious, how does this governance rules apply to `etcd-io/bbolt`?

Reply:
As the maintainers discussed, we will add separate Maintainer file in bbolt, probably in raft as well. cc @tbg 

I think the roles enumerated in this doc should be applied to all projects under etcd-io, at least including bbolt and raft. WDYT? @hexfusion @mitake @ptabor @serathius @spzala @tbg